calcure/loaders.py

```diff
@@ -173,14 +173,23 @@ def load(self):
 
     def load_country(self, country):
         """Load list of holidays from 'holidays' module"""
+
+        def get_country_and_subdivision(country):
+            """Get country and subdivision, where encoded."""
+            if ":" in country:
+                return tuple(country.split(":"))
+            else:
+                return (country, None)
+
         try:
             import holidays as hl
             from holidays import registry
 
+            (country, subdivision) = get_country_and_subdivision(country)
```

Comment:
Prior to this line, `country` would be set to either something like `UnitedKingdom` or `UnitedKingdom:England`. After this line, `country` is redefined to `UnitedKingdom`  and `subdivision` is defined as `England`.